nncf/pruning/base_algo.py

```diff
@@ -331,6 +339,17 @@ def hook(grad, mask, dim=0):
                 partial_hook = update_wrapper(partial(hook, mask=mask), hook)
                 self._hooks.append(bias.register_hook(partial_hook))
 
+    def check_pruning_rate(self, params):
+        """
+        Check that set only one of pruning target params
+        """
+        pruning_target = params.get('pruning_target', None)
+        pruning_flops_target = params.get('pruning_flops_target', None)
+        if pruning_target and pruning_flops_target:
```

Comment:
What if none of the parameters is specified? We should raise an exception as well.

Reply:
@AlexKoff88 Why? We had default `pruning_target` = 0.5.

Reply:
Ok. I found that in the code below.